.storybook/config.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
+
 if (process.env.NODE_ENV !== 'production') {
-  const { whyDidYouUpdate } = require('why-did-you-update');
-  whyDidYouUpdate(React);
+  const whyDidYouRender = require('@welldone-software/why-did-you-render');
+  whyDidYouRender(React);
```

Comment:
This fixes a bug where there were two version of React present.

Reply:
It's no longer necessary, but it will save people lots of debugging time if they want to use hooks in this repo.